src/plugins/intel_cpu/src/nodes/executors/acl/acl_deconv.cpp

```diff
@@ -36,12 +36,16 @@ ACLDeconvTensorInfo getACLDeconvTensorInfo(const DeconvAttrs& deconvAttrs,
     TensorInfo dstTensorInfo = TensorInfo(shapeCast(dstDims), 1,
                                           precisionToAclDataType(dstDescs[0]->getPrecision()), getAclDataLayoutByMemoryDesc(dstDescs[0]));
 
-    unsigned int pad_l =
-            (deconvAttrs.paddingL.size() > 1) ? static_cast<unsigned int>(deconvAttrs.paddingL.at(1)) : static_cast<unsigned int>(deconvAttrs.paddingL.at(0));
-    unsigned int pad_r =
-            (deconvAttrs.paddingR.size() > 1) ? static_cast<unsigned int>(deconvAttrs.paddingR.at(1)) : static_cast<unsigned int>(deconvAttrs.paddingR.at(0));
-    unsigned int pad_t = static_cast<unsigned int>(deconvAttrs.paddingL.at(0));
-    unsigned int pad_b = static_cast<unsigned int>(deconvAttrs.paddingR.at(0));
+    int temp_pad_l = (deconvAttrs.paddingL.size() > 1) ? static_cast<int>(deconvAttrs.paddingL.at(1)) : static_cast<int>(deconvAttrs.paddingL.at(0));
+    int temp_pad_r = (deconvAttrs.paddingR.size() > 1) ? static_cast<int>(deconvAttrs.paddingR.at(1)) : static_cast<int>(deconvAttrs.paddingR.at(0));
+    int temp_pad_t = static_cast<int>(deconvAttrs.paddingL.at(0));
+    int temp_pad_b = static_cast<int>(deconvAttrs.paddingR.at(0));
+
+    unsigned int pad_l = temp_pad_l < 0 ? 0 : temp_pad_l;
```

Comment:
Just wondering if it is correct, that we can get negative padding here?
It seems the previous version was not expecting this at all.
Does it mean that the previous version was working incorrectly?
Do we have a test for it?

Reply:
Agreed to simply return isSupported=false for the negative paddings

Reply:
@EgorDuplensky  done